code/modules/mob/living/carbon/monkey/monkey.dm

```diff
@@ -105,7 +105,7 @@
 
 		update_muts=1
 
-	if(!istype(src, /mob/living/carbon/monkey/diona))
+	if(type == /mob/living/carbon/monkey)
```

Comment:
>making every monkey type that isn't the default monkey or diona or golem NOT HAVE A LANGUAGE

The languages for all the other sub-races are added in their respective `new()`.  Skrell monkeys get Skrellian, tajaran monkeys get Siik'tajr, etc. The presence of the `istype()` on this line meant that previously, any sub-race that wasn't a diona had the ability to speak monkey in addition to their racial language. Moreover, monkey would be their default language.

I didn't know that about the type variable, though. Would `type == "/mob/living/carbon/monkey"` work the way I need it to?